File: MY APPLICATION 34/main/java/com/example/myapplication/MainActivity2.java
package com.example.myapplication;

import androidx.appcompat.app.AppCompatActivity;
import androidx.fragment.app.Fragment;
import androidx.fragment.app.FragmentManager;
import androidx.fragment.app.FragmentTransaction;

import android.os.Bundle;
import android.view.View;
import android.widget.Button;

public class MainActivity2 extends AppCompatActivity
{
    Button btn1,btn2;

    @Override
    protected void onCreate(Bundle savedInstanceState)
    {

        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main2);


        btn1=findViewById(R.id.btn1);
        btn2=findViewById(R.id.btn2);

        SecondFragment f1 =new SecondFragment();
        ThFragment i1= new ThFragment();

        btn1.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {



                replacer2(f1);


            }
        });

       btn2.setOnClickListener(new View.OnClickListener() {
           @Override
           public void onClick(View view)
           {
               replacer(i1);

           }
       });


    }

    void replacer(Fragment fr){
        FragmentManager fm=getSupportFragmentManager();
        FragmentTransaction ft=fm.beginTransaction();
        ft.replace(R.id.frm,fr).commit();

    }
    void replacer2(Fragment fr){
        FragmentManager fm=getSupportFragmentManager();
        FragmentTransaction ft=fm.beginTransaction();
        ft.replace(R.id.frm1,fr).commit();

    }
}

File: MY APPLICATION 19/main/java/com/example/myapplication/MainActivity2.kt
package com.example.myapplication

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import androidx.fragment.app.FragmentManager
import androidx.fragment.app.FragmentTransaction
import com.example.materialex.FirstFragment
import com.example.materialex.SecondFragment

class MainActivity2 : AppCompatActivity()
{
    override fun onCreate(savedInstanceState: Bundle?)
    {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main2)

        var f=FirstFragment()
        var fm:FragmentManager=supportFragmentManager
        var ft:FragmentTransaction=fm.beginTransaction()
        ft.replace(R.id.frmid,f).commit()

        //fragment show
        var f2=SecondFragment()
        var fm2:FragmentManager=supportFragmentManager
        var ft2:FragmentTransaction=fm2.beginTransaction()
        ft2.replace(R.id.frmid2,f2).commit()


    }
}
